pkg/tsdb/cloudwatch/cloudwatch.go

```diff
@@ -187,20 +193,12 @@ func (e *cloudWatchExecutor) getCWClient(region string) (cloudwatchiface.CloudWa
 }
 
 func (e *cloudWatchExecutor) getCWLogsClient(region string) (cloudwatchlogsiface.CloudWatchLogsAPI, error) {
-	e.mtx.Lock()
-	defer e.mtx.Unlock()
-
-	if logsClient, ok := e.logsClientsByRegion[region]; ok {
-		return logsClient, nil
-	}
```

Comment:
This change of no longer caching the logs client has an effect also on users not enabling the feature flag. Why is this change made? Will it have a negative effect on performance?

Reply:
I described why this change was made [here](https://github.com/grafana/grafana/pull/28176#discussion_r504032077)
I don't foresee a negative effect on performance as, as far as I can tell, the caching wasn't functioning correctly anyway, since a new executor was created for each request.

Reply:
Aha, thanks for pointing me to the explanation. I'm still unsure if the caching should be removed via this PR though, as it's for a completely different topic. I'll discuss it with the others.